README.md

```diff
@@ -10,41 +10,25 @@ Please see the [Demo Docker Repository](https://hub.docker.com/r/rapidsai/rapids
 
 ## Install cuDF
 
-### Conda
+#### Conda
 
-You can get a minimal conda installation with [Miniconda](https://conda.io/miniconda.html) or get the full installation with [Anaconda](https://www.anaconda.com/download).
+It is easy to install cuDF using conda. You can get a minimal conda installation with [Miniconda](https://conda.io/miniconda.html) or get the full installation with [Anaconda](https://www.anaconda.com/download).
 
-You can install and update cuDF using the conda command:
+Install and update cuDF using the conda command:
 
 ```bash
 conda install -c nvidia -c rapidsai -c numba -c conda-forge -c defaults cudf=0.3.0
 ```
 
 Note: This conda installation only applies to Linux and Python versions 3.5/3.6.
 
-You can create and activate a development environment using the conda commands:
-
-```bash
-# create the conda environment (assuming in base `cudf` directory)
-$ conda env create --name cudf_dev --file conda/environments/dev_py35.yml
-# activate the environment
-$ source activate cudf_dev
-# when not using default arrow version 0.10.0, run
-$ conda install -c nvidia -c rapidsai -c numba -c conda-forge -c defaults pyarrow=$ARROW_VERSION
-```
-
-This installs the required `cmake`, `nvstrings`, `pyarrow` and other
-dependencies into the `cudf_dev` conda environment and activates it.
-
-### Pip
+#### Pip
 
 Support is coming soon, please use conda for the time being.
 
 ## Development Setup
 
-The following instructions are tested on Linux Ubuntu 16.04 & 18.04, to enable
-from source builds and development. Other operatings systems may be compatible,
-but are not currently supported.
+The following instructions are for developers and contributors to cuDF OSS development. These instructions are tested on Linux Ubuntu 16.04 & 18.04. Use these instructions to build cuDF from source and contribute to its development.  Other operatings systems may be compatible, but are not currently supported.
```

Comment:
I guess you are ignoring the preferred 80 column width limit?

Reply:
Ehheh sorry, when writing docs in particular I tend to just put it all on one line, LaTeX style. I expect we're using PEP8? 80 columns works, I haven't seen any specific requests for it. I am, as you can see, a columnar width ignorer. My display goes up to 479 so when I need it wide I use a bigger window. :) I'll try to keep 80 columns in mind.

Reply:
I tend not to use any column width for written word because frequently a single word change in a paragraph makes very annoying cascading indentation changes for the rest of the paragraph.

Reply:
Actually that's a good point. I was just thinking of all the flake8 fixes I had to make. But that's only .py files. For .md you are right -- having to manually wrap is a pain, let's not do it. Ignore my comment!